src/woo_search/search_index/tasks.py

```diff
@@ -232,7 +234,10 @@ def index_publication(
     verkorte_titel: str,
     omschrijving: str,
     registratiedatum: datetime,
+    gepubliceerd_op: datetime,
```

Comment:
```suggestion
    gepubliceerd_op: datetime | None,
```
for backwards compatibiity